paasta_tools/cli/cmds/spark_run.py

```diff
@@ -321,6 +321,13 @@ def add_subparser(subparsers):
         default=False,
     )
 
+    list_parser.add_argument(
+        "--use-eks",
+        help="Use the EKS version of the target cluster rather than the Yelp-managed target cluster",
```

Comment:
[Nit, Optional] When I read this, it sounds to me that Spark Job uses EKS version of target k8s cluster instead of Yelp-managed target cluster. That isn't true, right? It's only the control plane/k8s masters which are the EKS version? 
"Use the EKS version of the target k8s master to connect to, rather than the Yelp-managed target cluster"

Reply:
No, we will be creating a new cluster on EKS; nodes cannot be a part of multiple kubernetes cluster and as such, we will be creating a new cluster with the same expected spark pools.  However, we hope to be able to obscure/omit the EKS cluster name from the command when we get to the point where it can be made the default.

Reply:
> No, we will be creating a new cluster on EKS; nodes cannot be a part of multiple kubernetes cluster and as such, we will be creating a new cluster with the same expected spark pools. However, we hope to be able to obscure/omit the EKS cluster name from the command when we get to the point where it can be made the default.

ack. 

IUC, the workflow is:
* Phase-1 now: k8s control plane is on EKS. 
* Phase-2: Spark pools will be on EKS (when is it expected).
The above explanation caters towards Phase-2, right?

Reply:
correct - we'll do the pool/cluster creation once we're ready to begin moving things (we can probably test this on our test cluster/pool with some real jobs once this is merged tho)

for now, the only folks use --use-eks will likely be us :)